lua/wire/client/sound_browser.lua

```diff
@@ -30,6 +30,39 @@ local TranslateCHAN = {
 	[CHAN_USER_BASE] = "CHAN_USER_BASE"
 }
 
+local TranslateGameMounts = {
```

Comment:
You can use the title from engine.GetGames instead can't you?

Reply:
Yeah, true. I just figured passing the additional string might have been weird if there’s only ever going to be this many games mountable to gmod. I’ll look at it when I get back from work.